src/components/MultipleDatePicker.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import Button from './Button';
+import DayPicker, { DateUtils } from 'react-day-picker/DayPicker';
+import 'react-day-picker/lib/style.css';
+import styled from 'styled-components';
+import { observable, action } from 'mobx';
+import { observer } from 'mobx-react';
+import { Calendar } from './Icons';
+
+@observer
+class MultipleDatePicker extends Component {
+  @observable show = false
+  @observable current = new Date()
+
+  @action.bound YearMonthForm = ({ date, localeUtils }) => {
+    const currentYear = new Date().getFullYear()
+    const fromMonth = new Date(currentYear - 1, 0)
+    const toMonth = new Date(currentYear + 5, 11)
+
+    const months = localeUtils.getMonths()
+    const years = []
+    for (let i = fromMonth.getFullYear(); i <= toMonth.getFullYear(); i += 1) {
+      years.push(i)
+    }
+
+    const handleChange = (e) => {
+      const { year, month } = e.target.form
+      this.current = new Date(year.value, month.value)
+    }
+
+    return (
+      <div className="DayPicker-Caption">
+        <select name="month" onChange={handleChange} value={date.getMonth()}>
+          {months.map((month, i) => (
+            <option key={month} value={i}>
+              {month}
+            </option>
+          ))}
+        </select>
+        <select name="year" onChange={handleChange} value={date.getFullYear()}>
+          {years.map(year => (
+            <option key={year} value={year}>
+              {year}
+            </option>
+          ))}
+        </select>
+      </div>
+    )
+  }
+
+  @action.bound showCalendar = (event) => {
+    event.preventDefault()
+    this.show = true
+  }
+
+  @action.bound closeCalendar = (event) => {
+    event.preventDefault()
+    this.show = false
+  }
+
+  @action.bound handleDayClick = (day, { selected }) => {
+    const { value } = this.props
```

Comment:
Props shouldn't be changed directly like this. Have the parent component pass an onChange-like listener (ex: `onDayClicked`) and call that function with the new value.